core/system/src/unvic.c

```diff
@@ -604,4 +563,12 @@ void unvic_init(void)
      * by IRQ number. For example, IRQ 0 has precedence over IRQ 1 if both have
      * the same priority level. */
     NVIC_SetPriorityGrouping(0);
+
+    /* Copy the user interrupt table into the handlers. */
+    for (uint32_t ii=0; ii < NVIC_VECTORS; ii++) {
+        g_unvic_vector[ii].id = UVISOR_BOX_ID_INVALID;
+        g_unvic_vector[ii].hdlr = (TIsrVector) user_vtor[ii + NVIC_OFFSET];
+        /* set default priority (SVC must always be higher) */
```

Comment:
Use complete sentences (with capital letter and correct punctuation) for new comments.